### PR TITLE
Update Slurm scripts to run benchmark via single Slurm job

### LINKED ISSUE #182: Update and verify single slurm job script
Update `run.sh` to ensure success running gpu-bdb multi-node with a single slurm job

### PR DESCRIPTION
**Pull request by ayushdg**
1. This PR updates the slurm runner scripts to make use of the `SLURM_NODEID` variable to decide scheduler & worker nodes within the same slurm job.

2. The PR also adds a simple status_file in a shared directory to poll the status of the job and end when the job completes. 

Both these updates should potentially simplify using gpu-bdb with `sbatch` and remove the need to manually kill jobs. 

Closes #182

### CODE REVIEW

File: gpu_bdb/benchmark_runner/slurm/run_bench.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+set -e pipefail
```
Comment (ayushdg) [inline gpu_bdb/benchmark_runner/slurm/run_bench.sh:1]: This is a combination of the `scheduler-client` and `spawn-workers` scripts which can be removed.

File: gpu_bdb/benchmark_runner/slurm/run.sh [hunk 2]
```diff
@@ -16,27 +15,9 @@ rm -rf $HOME/dask-local-directory/*
 srun \
     --account $ACCOUNT \
     --partition $PARTITION \
-    --nodes 1 \
-    --job-name gpubdb-sched \
+    --nodes $NODES \
+    --job-name ${ACCOUNT}-gpubdb:run_bench \
```
Comment (beckernick) [inline gpu_bdb/benchmark_runner/slurm/run.sh:19]: thanks for this

File: gpu_bdb/benchmark_runner/slurm/run_bench.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+until grep -q "FINISHED" "${STATUS_FILE}"
+do
+    sleep 30
+done
+
+pkill dask
+
+
+
+
```
Comment (beckernick) [inline gpu_bdb/benchmark_runner/slurm/run_bench.sh:55]:
```suggestion
```

**Review by beckernick — CHANGES_REQUESTED** [review]
This is great. Thank you for going through the Slurm docs for the NODEID. Can you also remove the newly unnecessary bash scripts?

**Review by beckernick — APPROVED** [review]
LGTM!

**Pull request merged**